use wxgtk2-2.4-config, wx-config is part of wx25
- Sebastian Czech <t2_@arcor.de>
--- audacity-src-1.2.3_orig/configure.in	2005-03-31 23:00:44.453246368 +0200
+++ audacity-src-1.2.3/configure.in	2005-03-31 23:07:55.473721296 +0200
@@ -516,7 +516,7 @@
 dnl --- check for required libraries ---
 
 dnl wxWindows -- we assume that if wx-config is found, wxWindows is successfully installed.
-AC_PATH_PROG(WX_CONFIG, wx-config, no, $PATH:/usr/local/bin )
+AC_PATH_PROG(WX_CONFIG, wxgtk2-2.4-config, no, $PATH:/usr/local/bin )
 if [[ "$WX_CONFIG" = "no" ]] ; then
 	AC_MSG_ERROR("Could not find wx-config: is wxWindows installed? is wx-config in your path?")
 fi
